Monet/ControllersViews/user_manag_sys3/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+class UsersController < ApplicationController
+  def index
+  	@users = User.all
+  end
+  def new
+  	@user = User.new	
+  end
+  def create
+  	@new_user = User.create(user_params)
+  	if @new_user.errors.any?
+  		flash[:error] = @new_user.errors.full_messages
+  		redirect_to "/users/new"
+  	else
+    flash[:success] = "Success - a new User!"
+    redirect_to "/users"
+    end
+  end
+  def show
+    @user = User.find(params[:id])
```

Comment:
Quick tip, since you're getting @user = User.find(params[:id]  several times, you could also just make a before_action that calls a private method to retrieve the user before edit/update/destroy run.